packages/protocol/test/governance/release_gold.ts

```diff
@@ -0,0 +1,1708 @@
+import { CeloContractName } from '@celo/protocol/lib/registry-utils'
```

Comment:
Can you rename this to `vesting.ts` just to make the review easier? Then, once it's approved, we can switch to the proper name